transformer.py

```diff
@@ -98,6 +106,20 @@ def transform_single_file(file_path, output_path):
             errors = f.read()
             raise RuntimeError("Failed to transform input file " + file_path + ": " + reason_bad + ' -- errors: \n' + errors)
 
+    flat_file = uproot.open(output_path)
+    flat_tree_name = flat_file.keys()[0]
+    attr_name_list = flat_file[flat_tree_name].keys()
+
+    arrow_writer = ArrowWriter(file_format=args.result_format, servicex=servicex,
```

Comment:
I think we need to preserve the ability to upload flat root files to object-store. This function will always turn the root file into arrow. Take a look at the old code to see how to upload the result directly to object-store